backend/danswer/chat/chat_utils.py

```diff
@@ -296,8 +305,19 @@ def get_chunks_for_qa(
 
             # We calculate it live in case the user uses a different LLM + tokenizer
             chunk_token = check_number_of_tokens(chunk.content)
+            if chunk_token > DOC_EMBEDDING_CONTEXT_SIZE + token_leeway:
```

Comment:
UCSD related issue change

Reply:
Shouldn't we only do this if `not latest_batch_indices`?